context/brand-context/default/scss/10-settings/buttons.scss

```diff
@@ -16,6 +16,7 @@ $context--button-font-size-normal: 1.6rem;
 $context--button-font-size-large: 1.8rem;
 $context--button-icon-margin: spacing(8);
 $context--button-padding: spacing(8);
+$context--button-padding-small: 0;
```

Comment:
Previous implementation of `@button-small` did not change the padding, so this is going to break any previous use of this. The `u-button-small` class is used in combination with the `u-button` class, so previously padding would come from the use of `u-button`. Make this value `inherit` and override (as you have) in brand settings to make this a non-breaking change